tests/Unit/ParallelAlgorithms/DiscontinuousGalerkin/Test_InitializeMortars.cpp

```diff
@@ -53,14 +52,13 @@ struct ElementArray {
   using metavariables = Metavariables;
   using chare_type = ActionTesting::MockArrayChare;
   using array_index = ElementIndex<Dim>;
-  using const_global_cache_tag_list = tmpl::list<>;
-  using add_options_to_databox = Parallel::AddNoOptionsToDataBox;
+  using const_global_cache_tag_list =
+      tmpl::list<::Tags::Domain<Dim, Frame::Inertial>>;
```

Comment:
(picking this spot at random)
I don’t think you _need_ to put the domain into the cache in many of these tests. You can leave it as is in the test and get it from the DataBox in the action that is being tested. That would be cleaner anyway I think, since the action should’t care if it’s in the DataBox or the cache.